### PR TITLE
Adding link to MDA as requested by the Edge team

### PR DESCRIPTION
**Pull request by batamig**
Rendered article section for review:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2750#enterprise-security-and-compliance

This PR makes the following changes:
* Adds a "Session policies" link pointing to https://learn.microsoft.com/defender-cloud-apps/session-policy-aad#integration-with-microsoft-edge-for-business-preview (or rather, for now, commenting out the anchor portion of link).
* Reformats the list of links to separate feature names from the link text, and make link text match destination title|heading per corp-level style guide & product-level style guide.

AB#45804003

### CODE REVIEW

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Approved

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -287,6 +287,7 @@ Microsoft Edge supports a variety of first-party enterprise security and complia
 * [Windows Information Protection](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip)
 * [Microsoft Defender Application Guard](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview)
 * [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about)
+* [Microsoft Defender for Cloud Applications](/defender-cloud-apps/what-is-defender-for-cloud-apps)
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:290]:
```suggestion
* [Microsoft Defender for Cloud App](/defender-cloud-apps/what-is-defender-for-cloud-apps)
```
https://learn.microsoft.com/en-us/defender-cloud-apps/what-is-defender-for-cloud-apps
Actual destination page title: "Microsoft Defender for Cloud Apps overview"
# hits on "cloud applications" in that page: 0
# hits on "cloud apps" in that page: 34

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor pass.  requested changing link text from "applications" to "apps" per consistent use of that in destination article.

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -284,12 +284,19 @@ To view the history of your previous downloads, go to `edge://downloads`.  To cl
...
-* [Microsoft Endpoint Data Loss Prevention (DLP)](/microsoft-365/compliance/endpoint-dlp-learn-about)
-* [Microsoft Insider Risk Management](/microsoft-365/compliance/insider-risk-management-solution-overview) 
-* [Microsoft Edge management service](/deployedge/microsoft-edge-management-service) 
 
+| Feature | Article |
+|---|---|
+| Windows Information Protection | [Protect your enterprise data using Windows Information Protection (WIP)](/windows/security/information-protection/windows-information-protection/protect-enterprise-data-using-wip) |
+| Microsoft Defender Application Guard | [Microsoft Defender Application Guard overview](/windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview) |
+| Microsoft Endpoint Data Loss Prevention (DLP) | [Learn about Endpoint data loss prevention](/microsoft-365/compliance/endpoint-dlp-learn-about) |
+| Microsoft Defender for Cloud Apps | [Session policies](/defender-cloud-apps/session-policy-aad) |
```
Comment (batamig) [inline microsoft-edge/privacy-whitepaper/index.md:293]:
```suggestion
| Microsoft Defender for Cloud Apps (Preview) | [Session policies](/defender-cloud-apps/session-policy-aad) |
```

**Comment by batamig** [comment]
#sign-off

**Review by jlam3 — APPROVED** [review]

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Approved

**Pull request merged**